### PR TITLE
updated french translation

### PR DESCRIPTION
**Pull request by SergiyGolov**
#130

### CODE REVIEW

File: resources/lang/fre.ini [hunk 1]
```diff
@@ -7,10 +7,78 @@ error = "Quelque chose s'est mal passé"
 noperm = "Vous n'avez pas la permission d'exécuter cette commande"
```
Comment (inxomnyaa) [inline resources/lang/fre.ini]: Please make sure to remove entries that are not present in https://github.com/thebigsmileXD/MagicWE2/blob/translations/resources/lang/eng.ini

**Review by inxomnyaa — CHANGES_REQUESTED** [review]

File: resources/lang/fre.ini [hunk 2]
```diff
@@ -1,24 +1,85 @@
...
+enabled = "activé"
+disabled = "désactivé"
+; errors
+error.command-error = "Il semble qu'il manque un argument ou que vous avez mal utilisé la commande!"
+error.runingame = "Veuillez éxécuter cette commande en jeu!"
+error.limitexceeded = "Vous êtes en train d'essayer de modifier trop de blocs d'un coup, réduisez la sélection ou augmentez la limite"
+error.notarget = "Bloc de cible pas trouvé. Augmentez la portée de l'outil avec //setrange si besoin"
+error.noselection = "Aucune sélection trouvée - sélectionnez une zone d'abord"
+error.selectioninvalid = "Sélection non valide ! Vérifiez que toutes les positions sont définis"
+error.nosession = "Aucune session s'est crée - vous n'avez probablement pas le droit d'utiliser {%0}"
```
Comment (Toinane) [inline resources/lang/fre.ini]:
```suggestion
error.nosession = "Aucune session n'a été créée - vous n'avez probablement pas le droit d'utiliser {%0}"
```

File: resources/lang/fre.ini [hunk 3]
```diff
@@ -1,24 +1,85 @@
...
 divider = "========================================"
 spacer = " -=+=- {%0} -=+=- "
 error = "Quelque chose s'est mal passé"
-noperm = "Vous n'avez pas la permission d'exécuter cette commande"
-runingame = "Utilisez cette commande dans le jeu!"
 commands = "Commandes"
+enabled = "activé"
+disabled = "désactivé"
+; errors
+error.command-error = "Il semble qu'il manque un argument ou que vous avez mal utilisé la commande!"
```
Comment (Toinane) [inline resources/lang/fre.ini]:
```suggestion
error.command-error = "Il semble qu'il manque un argument ou que vous ayez mal utilisé la commande !"
```

File: resources/lang/fre.ini [hunk 4]
```diff
@@ -1,24 +1,85 @@
...
+error.runingame = "Veuillez éxécuter cette commande en jeu!"
```
Comment (Toinane) [inline resources/lang/fre.ini]:
```suggestion
error.runingame = "Veuillez éxécuter cette commande en jeu !"
```

File: resources/lang/fre.ini [hunk 5]
```diff
@@ -1,24 +1,85 @@
...
+error.limitexceeded = "Vous êtes en train d'essayer de modifier trop de blocs d'un coup, réduisez la sélection ou augmentez la limite"
+error.notarget = "Bloc de cible pas trouvé. Augmentez la portée de l'outil avec //setrange si besoin"
+error.noselection = "Aucune sélection trouvée - sélectionnez une zone d'abord"
+error.selectioninvalid = "Sélection non valide ! Vérifiez que toutes les positions sont définis"
```
Comment (Toinane) [inline resources/lang/fre.ini]:
```suggestion
error.selectioninvalid = "Sélection non valide ! Vérifiez que toutes les positions sont définies"
```

File: resources/lang/fre.ini [hunk 6]
```diff
@@ -1,24 +1,85 @@
...
+confirmation = "Confirmation"
+confirmation.yes = "Oui"
+confirmation.no = "Non"
+; wand tool
+tool.wand = "Baguette"
+tool.wand.lore.1 = "Clic gauche sur un bloc pour définir position 1 d'une sélection"
+tool.wand.lore.2 = "Clic droit sur un bloc pour définir position 2 d'une sélection"
+tool.wand.lore.3 = "Utilisez //togglewand pour basculer sa fonctionnalité"
+tool.wand.disabled = "L'outil de baguette est desactivé. Utilisez //togglewand pour le réactiver"
+tool.wand.setenabled = "L'outil de vaguette est maintenant {%0}!"
```
Comment (Toinane) [inline resources/lang/fre.ini]:
```suggestion
tool.wand.setenabled = "L'outil de baguette est maintenant {%0}!"
```

**Review by Toinane — COMMENTED** [review]
I found some mistakes to correct. I corrected them in my comments

**Review by Toinane — APPROVED** [review]
As I'm french, these translations seems to be good for me!

**Review by inxomnyaa — APPROVED** [review]

**Comment by inxomnyaa** [comment]
Thanks! 💯

**Pull request merged**